quarkus/server/src/main/resources/META-INF/keycloak-server.json

```diff
@@ -1,12 +1,19 @@
 {
```

Comment:
Changes to this file is not necessary as all these will be possible to set with environment variables when we have the config work added.

Reply:
Yes, but not at the moment. That is the reason.

Reply:
In fact, that file has also config that is deprecated, like the request hostname provider. I didn't change that because of the config work. But for now, it helps to check current behavior and how it will look like once we have the config system in.